.travis.yml

```diff
@@ -1,10 +0,0 @@
-language: objective-c
-osx_image: xcode9
```

Comment:
Is there a way to continue building on Xcode 9 @NickEntin? Xcode 9 is the oldest Xcode this repo currently supports. It'd be good to continue at least ensuring that this repo compiles on that Xcode version.

Reply:
I can leave this config to keep building the framework on Travis using Xcode 9 as long as we remove the line to build the sample project. As is, the sample project doesn't build with Xcode 11, and I'd rather get that working.

Reply:
That works for me!

Reply:
Addressed in 53a7eb3

Reply:
Wait, nevermind, forgot we decided that dropping Xcode 9 support was fine. Too many commits and comments on this PR. 😅